Makefile

```diff
@@ -297,6 +299,7 @@ runs:
 
 sync-app:
 	aws s3 sync ./terraform/build/app s3://$(APP_SRC_BUCKET) --delete
+	aws s3 sync ./terraform/build/api s3://$(API_SRC_BUCKET) --delete
```

Comment:
The task is `sync-app` and it shouldn't delete API.
Unlike app, api.zip is just one file so I doubt we need to delete it.
Anyway, we need to define how to upload it. We've done it by running `aws lambda update-function-code`. Now we might need two steps, first upload to s3 and update the function from s3.